tee-worker/ts-tests/parachain-api/.gitignore

```diff
@@ -0,0 +1,3 @@
+.yarn
+node_modules
+dist/build
```

Comment:
Iffy on this one; I think it should be either all of `dist` or none of it, no?

Reply:
After testing, we do need change the `dist/src/index.d.ts` file.

Reply:
Yeah but then shouldn't we commit the whole `dist` directory as-is?

Reply:
Interesting. I have a feeling we'll be revisiting this in #1881 :thinking: